patches/nginx__1.15.8_dynamic_tls_records.patch

```diff
@@ -0,0 +1,270 @@
+From 696fddd15cd6a5a7b911557506fb5b1ab58e9b1d Mon Sep 17 00:00:00 2001
+From: Leon Klingele <git@leonklingele.de>
```

Comment:
btw, does this need to be Vlad's name (author of the [original patch](https://github.com/cloudflare/sslconfig/commit/554f0ce5d4a0a5e08d2b61b7eaa01fa6f7c3a228#diff-61c9f2d2bcb7fe2b22eb9b271eb842e2)), my name, or both?

Reply:
Hello.

This does not affect the patch.
That content is a comment.

Reply:
Hi, not exactly. It is the name used by Git as the `Author` of the commit.
Who is the proper `Author` of the commit? It's Vlad's work, which was rebased by me onto a newer release of nginx.

Reply:
You can read the [LICENSE](https://github.com/cloudflare/sslconfig/blob/master/LICENSE) information.